.buildkite/scripts/steps/functional/security_serverless_playwright.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/steps/functional/common.sh
+
+export JOB=kibana-security-solution-playwright
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+echo "---SERVERLESS - Security Solution Playwright Tests"
+
+yarn playwright install
+
+cd x-pack/test/security_solution_playwright
+
+set +e
```

Comment:
What's the purpose of `set +e` here?

Reply:
@jbudz I was replicating what we have in all our scripts.

Reply:
Why is it needed here?  The only command run is `yarn run:serverless`

Reply:
Removing it now.